ci/aws-iam-check-keys/requirements.txt

```diff
@@ -4,28 +4,40 @@
 #
 #    pip-compile
 #
-boto3==1.34.134
+--trusted-host pypi.python.org
```

Comment:
Not sure you want the --trusted-host settings here, Ben mentioned this on one of my commits